index.html

```diff
@@ -11,7 +11,7 @@ <h2 style="margin:0px">
           <div style="text-align:right; font-size:50%">Read more...</div>
           </a>
         </h2>
-        <h2>Current Version: <strong>4.0.1</strong>
```

Comment:
This should probably be included in .bumpversion.cfg then.

Reply:
Proposal is to redirect figure.o.org to www.o.org to reduce the maintenance